# Name of the agent publishing the octomap
string agent_name

# The actual octree msg
octomap_msgs/Octomap octomap
